projects/plugins/jetpack/extensions/blocks/subscriptions/subscriptions.php

```diff
@@ -1046,14 +1047,14 @@ function get_paywall_access_question( $post_access_level ) {
 			if ( $tier !== null ) {
 				return sprintf(
 				// translators:  Placeholder is the tier name
-					__( 'I am already a <i>%s</i> paid-subscriber', 'jetpack' ),
+					__( 'Already a <i>%s</i> paid subscriber?', 'jetpack' ),
```

Comment:
We talked with @crisbusquets to drop the tier name in the mid-sentence out, but didn't actually come up with alternative that I remember.

Reply:
This was the last solution we discussed:

Subscriber-only paywall -> Already a subscriber? Log in.
Paid-subscriber only paywall -> Already a paid subscriber? Log in.
Higher tier paywall -> Already a higher-tier paid subscriber? Log in.

The 'Log in' is the link. Or maybe everything could be clickable. What do you think?

Reply:
Nice, thanks! 👍 

Let's drop "Log in" as pre-caution to issues Fab and Miguel were raising, make sure that all sentences are question format (that makes it more actionable rather than without question mark like it's in production) and make the whole sentence clickable/underlined.